src/updated_guidance.rs

```diff
@@ -0,0 +1,258 @@
+use ixa::{
+    define_derived_property, define_person_property_with_default, define_rng, trace, Context,
+    ContextPeopleExt, ContextRandomExt, PersonPropertyChangeEvent,
+};
+
+use statrs::distribution::Uniform;
+
+use crate::{
+    infectiousness_manager::InfectionStatusValue,
+    interventions::ContextTransmissionModifierExt,
+    parameters::ContextParametersExt,
+    symptom_progression::{SymptomValue, Symptoms},
+};
+
+define_person_property_with_default!(MaskingStatus, bool, false);
+define_person_property_with_default!(IsolatingStatus, bool, false);
+define_derived_property!(Symptomatic, Option<bool>, [Symptoms], |data| match data {
```

Comment:
```suggestion
define_derived_property!(PresentingWithSymptoms, Option<bool>, [Symptoms], |data| match data {
```

because a presymptomatic person will become symptomatic.

Reply:
Also, why not have this just be a bool? Not sure I see the use case for the `Some(false)` variant.